bottle-configs/aws-sam-cli-nightly.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "1.21.1.dev202101010000",
```

Comment:
will this be automated by a bot?

Reply:
yes, it is a placeholder now. The sam-cli bot will update it once source code is tagged.